teuthology/provision.py

```diff
@@ -195,6 +200,168 @@ def __del__(self):
         self.remove_config()
 
 
+class ProvisionOpenStack(OpenStack):
```

Comment:
Do you think it would be feasible/desirable to implement an abstract base ``Provision`` class that ``Downburst`` and ``ProvisionOpenStack`` (should maybe just be called ``OpenStack``) could inherit from? 

Just to formalize a bit more that they intend to provide a similar interface: ``create()``, ``destroy()``, ``build_config()``, ``remove_config()`` - though the last two are currently implemented by differently-named methods in ``ProvisionOpenStack``